pkg/cindervolume/volumes.go

```diff
@@ -33,29 +37,35 @@ func GetVolumes(parentName string, name string) []corev1.Volume {
 		},
 	}
 
-	return append(cinder.GetVolumes(parentName, true), volumeVolumes...)
+	// Set the propagation levels for CinderVolume, including the backend name
+	propagation := []storage.PropagationType{cinder.Cinder, cinder.CinderVolume, storage.PropagationType(strings.TrimPrefix(name, "cinder-volume-"))}
```

Comment:
Same comment about making storage.PropagationType a constant.

Reply:
Done, we have a basic propagation over `Cinder` (global) and `CinderVolume` (group), and we can `append` the instance to the variable because it's dynamically generated.